src/schemas/json/workflows.json

```
@@ -271,6 +274,38 @@
           "type": "object"
         }
       ]
+    },
+    "for": {
+      "description": "Iterate over a sequence of numbers or through a collection of data, such as a list or map.",
```

```suggestion
      "description": "Iterates over a sequence of numbers or through a collection of data\nhttps://cloud.google.com/workflows/docs/reference/syntax/syntax-search",
```

Optional change: reference docs instead of copying description completely.